src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -24,7 +25,18 @@
 from sentry.utils import json, metrics
 from sentry.utils.outcomes import Outcome, track_outcome
 
+MOBILE_EVENT_SAMPLE_RATE = 0.5
+RRWEB_EVENT_COUNT_SAMPLE_RATE = 0.5
+
 logger = logging.getLogger("sentry.replays")
+logger.addFilter(
+    SamplingFilter(
+        {
+            "mobile_event": MOBILE_EVENT_SAMPLE_RATE,
+            "rrweb_event_count": RRWEB_EVENT_COUNT_SAMPLE_RATE,
```

Comment:
Are either of these log messages emitted in our consumer?

Reply:
Do you mean a different file?

Reply:
Just realized I was looking at the `event_type` in `extras`. The msg is `sentry.replays.slow_click`. I'll have to rethink this

Reply:
Do we want different rates for these, or could we sample all slow click logs at the same rate for now?

Reply:
Same rate is fine.

Reply:
Can you access the `extra` field within your sampler class?

Reply:
I can. The filter will become specific to our usecase though, so I'd move it into the __init__ file